schedule/yast/dud_development_tools.yaml

```diff
@@ -31,3 +31,12 @@ schedule:
   - installation/reboot_after_installation
   - installation/grub_test
   - installation/first_boot
+  - console/validate_dud_addon_repos
+  - console/orphaned_packages_check
+test_data:
+  dud_repos:
+    - URI: 'ftp://openqa.suse.de/SLE-15-Module-Development-Tools-POOL-x86_64-Media1-CURRENT'
```

Comment:
Why do we need `-` here? I guess we need it as a hash key here.
Another idea, could you please add comment to point to dud file we use as it's url defined in it. I'm also wondering if we should update it for each SP, but as long as we don't install anything from it, it should be fine.

Reply:
Here I added it as array in order to be able to reuse the test module in case we'll need to check several repositories in one test. So, it was for purpose to make the test as much flexible as possible. If you think we'll not have it potentially that several repos should be checked in one test, then I can remove that part.

Reply:
I guess than we should make key `URIs` and make list as a value for it? I would not mix arrays and hashes, tbh.

Reply:
I made it by analogy with RAID tests. I'm sorry, but I didn't get what is the problem of using array of hashes?

Reply:
Ah, sorry, I've misread yaml syntax, but in the code I see that we have array of hashes here and not mix of both, so nevermind.